cmd/api/handler/validators.go

```diff
@@ -77,3 +81,20 @@ func msgTypeValidator() validator.Func {
 		return err == nil
 	}
 }
+
+func isNamespace(s string) bool {
+	hash, err := base64.StdEncoding.DecodeString(s)
+	if err != nil {
+		return false
+	}
+	if len(hash) != 29 {
```

Comment:
mb create a constant with this value or move the hash by length validation logic to a namespace related function? We already have 3 places with this kind of logic.